python/doc/examples/functional_modeling/field_functions/plot_process_sample.py

```diff
@@ -0,0 +1,165 @@
+
+# %%
+# Let's visualize this sample of the field.
+
+# %%
+graph = ot.Graph("One realization of the stochastic process", "Strain", "Stress (Pa)", True, "")
+curve = ot.Curve(vertices.asPoint(), Y)
+graph.add(curve)
+view = viewer.View(graph)
+
+# %%
+# The distribution of the input vector :math:`\vect{X}` is defined.
+
+# %%
+R_distribution = ot.LogNormal()
+R_distribution.setParameter(ot.LogNormalMuSigma()([750e6, 11e6, 0.0]))
+C_distribution = ot.Normal(2750e6, 250e6)
+gamma_distribution = ot.Normal(10., 2.)
+X_distribution = ot.JointDistribution([R_distribution, C_distribution, gamma_distribution])
+
+# %%
+# Generate different samples of the field
+# ---------------------------------------
+
+# %%
+# 50 samples of the input vector :math:`\vect{X}` are generated.
+
+# %%
+n_samples = 50
+X_samples = X_distribution.getSample(n_samples)
+
+# %%
+# The Chaboche model is evaluated for each of the input vector samples.
+
+# %%
+Y_list = []
+for i in range(n_samples):
+    f = ot.ParametricFunction(cm.model, X_indices, X_samples[i, :])
+    Y = [[f(vertices[i])[0]] for i in range(N)]
+    Y_list.append(Y)
+
+# %%
+# Let's visualize the 50 resulting samples of the field.
+
+# %%
+graph = ot.Graph("Realizations of the stochastic process", "Strain", "Stress (Pa)", True, "")
+for i in range(n_samples):
+    curve = ot.Curve(vertices, Y_list[i])
+    curve.setColor("blue")
+    graph.add(curve)
+view = viewer.View(graph)
+
+# %%
+# Creation of the process sample
+# ------------------------------
+
+# %%
+# It is possible to create a :class:`~openturns.ProcessSample` from the obtained field samples.
+# For that, each obtained sample is added to the :class:`~openturns.ProcessSample` using the :class:`~openturns.Field` structure.
+# When the :class:`~openturns.ProcessSample` is created, by default a process sample with a value of 0.
+# for all the vertices is stored so it is important to remove it.
+
+# %%
+process_sample = ot.ProcessSample(mesh, n_samples, 1)
+process_sample.clear()
+for i in range(n_samples):
+    process_sample.add(ot.Field(mesh, Y_list[i]))
+
+# %%
+# It is then possible to compute different statistics on the :class:`~openturns.ProcessSample` such as the mean, median, variance, ...
+
+# %%
+process_sample_mean = process_sample.computeMean()
+process_sample_median = process_sample.computeMedian()
+process_sample_variance = process_sample.computeVariance()
+
+# %%
+# Let's visualize the mean of the process sample.
+
+# %%
+graph = ot.Graph("Sample process mean and realizations", "Strain", "Stress (Pa)", True, "")
+for i in range(n_samples):
+    if i == 0:
+        curve = ot.Curve(vertices, Y_list[i], "process samples")
+    else:
+        curve = ot.Curve(vertices, Y_list[i])
+    curve.setColor("blue")
+    graph.add(curve)
+
+curve = ot.Curve(vertices, process_sample_mean, "process sample mean")
+curve.setColor("red")
+graph.add(curve)
+graph.setLegendPosition("topleft")
+view = viewer.View(graph)
```

Comment:
# sphinx_gallery_thumbnail_number = ?

Reply:
The modifications discussed during CT have been made.
The use of "a parametric model taking epsilon as the single input and R, C, gamma as parameters, set to the true values" was already implemented in the existing examples.
Otherwise, all the modifications are done.